code/controllers/subsystem/blackbox.dm

```diff
@@ -174,3 +174,59 @@ SUBSYSTEM_DEF(blackbox)
 	key_type = new_key_type
 	version = new_version
 	json = list()
+
+
+/datum/controller/subsystem/blackbox/proc/ReportDeath(mob/living/L)
+	set waitfor = FALSE
+	if(sealed)
+		return
+	if(!L?.key || !L.mind)
+		return
+	var/sqlname = L.real_name
```

Comment:
```suggestion
	var/sqlname = sanitizeSQL(L.real_name)
```
And so on. No need to define the vars before the connect check, right?

Reply:
I was thinking about that, but i assume its because they want the check as late as possible?